### PR TITLE
install python3-policycoreutils for pulpcore on el8

### PR DESCRIPTION
**Pull request by evgeni**
pulp_installer uses the seport module, but doesn't install all of its
prerequisites, workaround that for now!

Real fix is in https://github.com/pulp/pulp_installer/pull/513/

### CODE REVIEW

**Comment by ekohl** [comment]
Given the linked PR already has an approval, I'm leaning to waiting for that to be merged.

**Comment by ekohl** [comment]
Linked PR is merged. Closing.

**Comment by evgeni** [comment]
There is no release yet, tho. And I need 3.9 built by wednesday, which this blocks.

**Review by ehelms — APPROVED** [review]
Add a TODO or file a Forklift issue to drop this later?

**Comment by evgeni** [comment]
done: https://github.com/theforeman/forklift/issues/1276

**Pull request merged**
